content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -143,6 +143,7 @@ For a reference to old feature gates that are removed, please refer to
 | `LoggingBetaOptions` | `true` | Beta | 1.24 | - |
 | `MatchLabelKeysInPodTopologySpread` | `false` | Alpha | 1.25 | 1.26 |
 | `MatchLabelKeysInPodTopologySpread` | `true` | Beta | 1.27 | - |
+| `MatchLabelKeysInPodAffinity` | `false` | Alpha | 1.29 | - |
```

Comment:
Please make sure to maintain alphabetical order when adding new data in the table.